dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/enums/Status.java

```diff
@@ -297,7 +297,8 @@
     NOT_SUPPORT_UPDATE_TASK_DEFINITION(50056, "task state does not support modification", "当前任务不支持修改"),
     NOT_SUPPORT_COPY_TASK_TYPE(50057, "task type [{0}] does not support copy", "不支持复制的任务类型[{0}]"),
     HDFS_NOT_STARTUP(60001, "hdfs not startup", "hdfs未启用"),
-
+    STORAGE_NOT_STARTUP(60002, "storage not startup", "存储未启用"),
+    STORAGE_CANNOT_RENAME(60003, "directory cannot be renamed", "S3无法重命名文件夹"),
```

Comment:
Is it only for `S3` or for all type of storage? if for all storage we should not use keyword `S3`

Reply:
had changed STORAGE_CANNOT_RENAME to S3_CANNOT_RENAME